src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -70,25 +70,29 @@ public static SerialNumber parseSerialNumber(String serialNumber) throws ParseEx
      *
      * @return Author object from remark string
      */
-    public static Author parseAuthor(String author) {
+    public static Author parseAuthor(String author) throws ParseException {
         requireNonNull(author);
         String trimmedAuthor = author.trim();
+        if (!Author.isValidAuthor(trimmedAuthor)) {
+            throw new ParseException(Author.MESSAGE_CONSTRAINTS);
+        }
         return new Author(trimmedAuthor);
     }
 
     /**
      * Parses a {@code String genre} into a {@code Genre}.
-     * Leading and trailing whitespaces will be trimmed.
+     * Leading and trailing whitespaces will be trimmed, and genre will be converted to UPPERCASE
```

Comment:
A good workaround to enable case insensitive search!

Reply:
In the end it took more effort than implementinig case-insensitive search directly haha